### ISSUE TITLE
Poor web panel accessibility for screen readers

### ISSUE THREAD

**Issue by cyrmax** [alireza0/x-ui#1399]
**Describe the bug**
The web panel has very poor accessibility for blind users who use screenreaders and don't see a screen.

**To Reproduce**
0. Download the free NVDA screenreader from nvaccess.org;
1. Launch it and open the panel;
2. Try to close your eyes and use the panel interface only with keyboard, without mouse and without looking at your screen;
3. You will see several major problems with accessibility;
4. Comboboxes (for language, for protocols, etc) are not accessible with keyboard and are not exposed to a screenreader. Consider using native <select> elements rather than something custom;
5. You will see that menu options have some "icon:something" text near every option and this is annoying to listen when you navigate with screenreader;
6. You may notice that some icons have no text description at all and are identified only by an icon which is announced like "graphics" or "icon I";

**Expected behavior**
The web panel interface should be usable with a screenreader and should conform to WCAG standard;

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Brave, Chrome, Edge
 - Version: Latest at versions at the moment of creation of this issue

**Smartphone (please complete the following information):**
n/a

**Additional context**
NVDA screenreader version 2024.1 is used
